src/bin/mdbook-epub.rs

```diff
@@ -39,7 +43,10 @@ fn run(args: &Args) -> Result<(), Error> {
     // get a `RenderContext`, either from stdin (because we're used as a plugin)
     // or by instrumenting MDBook directly (in standalone mode).
     let ctx: RenderContext = if args.standalone {
-        let md = MDBook::load(&args.root).map_err(SyncFailure::new)?;
+        let book_root_clone = args.root.clone();
+        let error = format!("book.toml root file is not found by a path {:?}",
+                        &book_root_clone.into_os_string().into_string().to_owned().unwrap());
```

Comment:
You can just use `args.root.display()` here. 

That'll let you avoid the unnecessary copies (there are two, one for `args.root.clone()` and another for `into_string().to_owned()`), and the panic from `unwrap()`.

Reply:
fixed to you version